test/utils/http_static_test_server.py

```diff
@@ -28,6 +29,7 @@ class HTTPStaticTestServer(object):
     """
 
     def __init__(self, port=8088):
+        HTTPServer.address_family = socket.AF_INET6
```

Comment:
This changes the address family of every instance of HTTPServer instantiated after this point, not just this one. While that's likely *zero* instances, this should probably be overridden in an HTTPServer subclass due to the inconsiderate design of python's SocketServer classes.